hugo/assets/scripts/arithmetics/arithmetics-tools.tsx

```diff
@@ -1,4 +1,4 @@
-import { monaco } from "monaco-editor-wrapper/.";
+import { monaco } from "langium-website-foundation/bundle";
```

Comment:
langium-website-core/bundle might be good here.

Reply:
Changed to `core` in next commit ⬇️